components/style/properties/gecko.mako.rs

```diff
@@ -1339,6 +1339,80 @@ fn static_assert() {
     ${impl_coord_copy('column_width', 'mColumnWidth')}
 </%self:impl_trait>
 
+<%self:impl_trait style_struct_name="Counters"
```

Comment:
Actually, could you split this commit out into a separate PR? I've got to run pretty soon and may not have time to review the whole thing, and it's pretty separate from the atom stuff (and also presumably needs some gecko-side changes).